### PR TITLE
More documentation and tests for Ruby version managers

### PR DESCRIPTION
**Pull request by chrisseaton**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,6 +3,9 @@
 A high performance implementation of the Ruby programming language. Built on the
 GraalVM by [Oracle Labs](https://labs.oracle.com).
 
+We recommend that you use a [Ruby version manager](doc/user/ruby-managers.md)
+to use TruffleRuby.
```
Comment (eregon) [inline README.md]: This should be under `Getting Started`.

File: test/truffle/docker/README.md [hunk 2]
```diff
@@ -24,3 +24,12 @@ using.
 
 Docker will need to run the container with at least 8 GB of RAM if you are using
 virtualisation, to give enough space for the native image to build.
+
+## Ruby Managers
+
+There are also tested by Dockerfiles. These all run on Ubuntu. We test rbenv,
```
Comment (eregon) [inline test/truffle/docker/README.md]: There => They but I think it'd be clearer to use 
Ruby Managers are also tested by Dockerfiles.

Comment (chrisseaton) [reply test/truffle/docker/README.md]: Fixed.

File: README.md (hunk 1 above)
Comment (chrisseaton) [reply README.md]: Fixed.

File: doc/user/ruby-managers.md [hunk 3]
```diff
@@ -9,6 +11,8 @@ directory of rbenv:
 
 ```bash
 ln -s path/to/graalvm/jre/languages/ruby "$RBENV_ROOT/versions/truffleruby"
+rbenv global truffleruby
```
Comment (nirvdrum) [inline doc/user/ruby-managers.md]: This is a bold suggestion.

Comment (chrisseaton) [reply doc/user/ruby-managers.md]: What would you use, `shell`?

Comment (eregon) [reply doc/user/ruby-managers.md]: Yes, seems better to me.

**Pull request merged**